src/Books.js

```diff
@@ -1,13 +1,58 @@
-import React from 'react';
-import BookList from './components/BookList';
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
 import NewBookForm from './components/NewBookForm';
+import {
+  fetchBooksAsync, addBookAsync, removeBookAsync, getBooks,
+} from './redux/books/BookSlice';
+import Book from './components/Book';
 
-const Books = () => (
-  <div>
-    <h2>Books</h2>
-    <BookList />
-    <NewBookForm />
-  </div>
-);
+const Books = () => {
+  const dispatch = useDispatch();
+  const books = useSelector(getBooks);
+  console.log(books);
```

Comment:
### [ OPTIONAL ]
- [ ] It will be great to ensure you remove the `console.log` statements. This is because they expose sensitive information about the application on the console.